perf/pinot/build.xml

```diff
@@ -35,17 +35,20 @@
 				<mkdir dir="${DIR_NAME}"/>
 				<echo message="Downloading Pinot"/>
 				<exec executable="wget" failonerror="true" dir="${DIR_NAME}">
-					<arg line="https://singhnitin.blob.core.windows.net/pinot-build/apache-pinot-0.12.1-bin.tar.gz"/>
+					<arg line="https://downloads.apache.org/pinot/apache-pinot-0.12.1/apache-pinot-0.12.1-src.tar.gz"/>
 				</exec>
 				<echo message="Extracting Pinot"/>
 				<exec executable="tar" failonerror="true" dir="${DIR_NAME}">
-					<arg line="-xvf apache-pinot-0.12.1-bin.tar.gz"/>
+					<arg line="-xvf apache-pinot-0.12.1-src.tar.gz"/>
 				</exec>
 			</else>
 		</if>
+		<echo message="Building Pinot in ${basedir}/${DIR_NAME}/apache-pinot-0.12.1-src/pinot-perf"/>

```

Comment:
Reason we are building from source as opposed to using the binary?

Reply:
OK, let's add that as a comment so future you remembers :-)

Reply:
Done.